components/x11/urxvt/pkg5

```diff
@@ -19,4 +19,4 @@
         "terminal/urxvt"
     ],
     "name": "rxvt-unicode"
```

Comment:
pkg5 misses the perl update

Reply:
The file was updated but for some reason it is this perl that is detected: I have run it twice again to check that I did not forget. I suspect something is not updated yet and that makes the dependency produced to the old one.

Reply:
On my system it is properly detected.

Reply:
Maybe I should update the system to a more recent build?